index.html

```diff
@@ -290,7 +291,55 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
-
+    <div class="contact-form">
+      <div class="form info">
+        <h4 class="form-description">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h4>
+      </div>
+      <div class="form form-fields">
+        <form action="https://formspree.io/f/mbjqgwwa" method="POST">
+          <div>
+            <input
+              type="text"
+              name="firstname"
+              id="firstname"
+              placeholder="First Name"
+              required
+            />
+            <input
+              type="text"
+              name="lastname"
+              id="lastname"
+              placeholder="Last Name"
+              required
+            />
```

Comment:
- [ ] I like how you indented and used correct types and attributes for the inputs excellent job about that, 
however, the name inputs in this case `Last name` should have `maxlength=30` this is to make sure that the users do not accidentally put a long paragraph instead of their name :smile: , kindly add that. :+1:  :grinning: .

Reply:
Done.